README.md

```diff
@@ -265,6 +265,61 @@ The static site that gets built is pushed to the `documentation` branch of this
 We host the site on netlify. On master branch builds (see `.github/workflows/documentation.yml`), we push the built docs to
 the `documentation` branch. Netlify automatically re-deploys the docs pages whenever there is a change to that branch.
 
+## Releases
+### Release Timeline
+**For Rasa Open Source, we usually commit to time-based releases, specifically on a monthly basis.** 
+This means that we commit beforehand to releasing a specific version of Rasa Open Source on a specific day, 
+and we cannot be 100% sure what will go in a release, because certain features may not be ready. 
+
+At the beginning of each quarter, the Rasa team will review the scheduled release dates for all products and make sure 
+they work for the projected work we have planned for the quarter, as well as work well across products.
+
+**Once the dates are settled upon, we update the respective milestones.**
+
+### Cutting a Major / Minor release
+#### A week before release day
+
+1. **Make sure the [milestone](https://github.com/RasaHQ/rasa/milestones) already exists and is scheduled for the 
```

Comment:
should we add the name of the release owner to the milestone description?

Reply:
Hmm I mean we can but it adds manual work, which I am usually against unless there's a good reason for it. Do you see a big benefit in doing this? 🤔

Reply:
It would be transparent for the community and also for Rasas who didn't follow the #product-engineering channel who is cutting the release.

Reply:
Hmmm I see your point. But does it bring benefit for the community to know the person who is cutting the release? (Also I figured all the Rasas were on #product-engineering ahah)

Reply:
They are all in there, but I e.g. don't how closely the research team or CSE is following the channel. 

>  But does it bring benefit for the community to know the person who is cutting the release?

👍 True.

Let's skip adding it for now and see how it goes 👍